core/domain/user_services_test.py

```diff
@@ -96,7 +96,7 @@ def test_get_usernames(self):
         user_ids = []
         for gae_uid, email, name in python_utils.ZIP(
                 gae_user_ids, user_emails, usernames):
-            if gae_uid != feconf.SYSTEM_COMMITTER_GAE_ID:
+            if gae_uid is not None:
```

Comment:
Should not have "if" statements in tests. (Not sure why you need this one anyway?)

Reply:
Fixed.